deploy/helm/secret-operator/crds/crds.yaml

```diff
@@ -58,6 +58,27 @@ spec:
 
                                 If `autoGenerate: true` then the Secret Operator will prepare a new CA certificate the old CA approaches expiration. If `autoGenerate: false` then the Secret Operator will log a warning instead.
                               type: string
+                            keyGeneration:
+                              default:
+                                rsa:
+                                  length: 2048
+                              description: The TLS algorithm and required configuration settings. Currently only RSA and key length of 2048, 3072 or 4096 bits can be configured.
```

Comment:
note: There is no "TLS algorithm". There is only the algorithm used to generate a keypair which is used to sign the certificate which is used by the server to provide identification (server name, CA, and public encryption key).

Reply:
Removed the TLS algorithm stuff.